closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurityProperties.java

```diff
@@ -0,0 +1,765 @@
+     * @param string Input string for checking
+     * @return true if the brackets are balanced
+     */
+    private boolean areBracketsBalanced(String string) {
+
+        Deque<Character> deque = new LinkedList<>();
+
+        for (char ch : string.toCharArray()) {
+            if (ch == '{' || ch == '[' || ch == '(') {
+                deque.addFirst(ch);
+            } else if (ch == '}' || ch == ']' || ch == ')') {
+                if (!deque.isEmpty()
+                        && ((deque.peekFirst() == '{' && ch == '}')
+                                || (deque.peekFirst() == '[' && ch == ']')
+                                || (deque.peekFirst() == '(' && ch == ')'))) {
+                    deque.removeFirst();
+                } else {
+                    return false;
+                }
+            }
+        }
+        return deque.isEmpty();
+    }
+
+    /**
+     * Nested class for provider's constraints
+     */
+    private static final class Constraint {
+        final String type;
+        final String algorithm;
+        final String attributes;
+
+        Constraint(String type, String algorithm, String attributes) {
+            super();
+            this.type = type;
+            this.algorithm = algorithm;
+            this.attributes = attributes;
+        }
+
+        String getType() {
+            return type;
+        }
+
+        String getAlgorithm() {
+            return algorithm;
+        }
+
+        String getAttributes() {
+            return attributes;
+        }
+    }
+
+    public String getDescName() {
+        return descName;
+    }
+
+    public String getDescNumber() {
+        return descNumber;
+    }
+
+    public String getDescPolicy() {
+        return descPolicy;
+    }
+
+    public String getDescSunsetDate() {
+        return descSunsetDate;
+    }
+
+    public String getJdkTlsDisabledNamedCurves() {
+        return jdkTlsDisabledNamedCurves;
+    }
+
+    public String getJdkTlsDisabledAlgorithms() {
+        return jdkTlsDisabledAlgorithms;
+    }
+
+    public String getJdkTlsDphemeralDHKeySize() {
+        return jdkTlsDphemeralDHKeySize;
+    }
+
+    public String getJdkTlsLegacyAlgorithms() {
+        return jdkTlsLegacyAlgorithms;
+    }
+
+    public String getJdkCertpathDisabledAlgorithms() {
+        return jdkCertpathDisabledAlgorithms;
+    }
+
+    public String getJdkSecurityLegacyAlgorithm() {
+        return jdkSecurityLegacyAlgorithm;
+    }
+
+    public String getKeyStoreType() {
+        return keyStoreType;
+    }
+
+    public String getKeyStore() {
+        return keyStore;
+    }
+
+    public List<String> getProviders() {
+        return providers;
+    }
+
+    public String getJdkSecureRandomProvider() {
+        return jdkSecureRandomProvider;
+    }
+
+    public String getJdkSecureRandomAlgorithm() {
+        return jdkSecureRandomAlgorithm;
+    }
+
+    public Map<String, Constraint[]> getProviderConstraints() {
+        return providerConstraints;
+    }
```

Comment:
This can't be much use outside this class: it involves a private class.

Reply:
Remove this method since no outside this class invoking needed.